### DISCUSSION TITLE
Push ARM powered docker image to dockerhub

### DISCUSSION THREAD

**Discussion by Mayurifag** [anycable/anycable#156] [Ideas]
If you have Apple Silicon powered Macbook (on arm), downloaded image will be executed with amd64 emulation. However, there is no problem to push the arm build, because its already is building on CI. Can we have an image? I hope for universal build, though, which I can launch on my arm and on my amd64 machines.

Thanks in advance! :)

**Comment by palkan** [comment]
Thanks for the feedback!
Will try to prepare one for the next release.

> I hope for universal build, though, which I can launch on my arm and on my amd64 machines.

If you know some tips of creating universal images for Go projects, I would appreciate that :) Haven't dug into this yet.

**Comment by palkan** [comment]
Okay, I pushed an experimental multi-arch edge image: https://hub.docker.com/layers/87292678/anycable/anycable-go/edge/images/sha256-874bcabccba1e1f57664d6160bb66b20c63723c186d7bbc3f23b4c6e1202bf46?context=repo

Could you please check it?

**Comment by Mayurifag** [comment]
I've had issues to download image via docker-compose recently, but now it seems to work fine 🔥 I've checked docker dashboard in macos, so it didnt show image works on amd64. However, didnt check if service works on all possible cases, but I think you didnt change much, so everything is fine. Hope to see this change in mainstream 👍🏻 

I've downloaded anycable:edge, didnt specify exact hash of docker image

**Reply by palkan** [reply]
Thanks! Will prepare new releases later this week.
